crates/sui-core/src/authority_batch.rs

```diff
@@ -160,6 +160,11 @@ impl crate::authority::AuthorityState {
             .next()
             .unwrap();
 
+        let metrics = self.metrics.clone();
```

Comment:
Here the code is only called once at node startup, right? Is that intended for this metric?